test/e2e/utils/helpers.go

```diff
@@ -116,7 +116,7 @@ func IsNodeConfigDoneWithNodeTuningFunc(nodes []*corev1.Node) func(nc *scyllav1a
 }
 
 func RolloutTimeoutForScyllaCluster(sc *scyllav1.ScyllaCluster) time.Duration {
-	return SyncTimeout + time.Duration(GetMemberCount(sc))*memberRolloutTimeout
+	return SyncTimeout + time.Duration(GetMemberCount(sc))*memberRolloutTimeout + cleanupJobTimeout
```

Comment:
shouldn't `cleanupJobTimeout` be added to the `memberRolloutTimeout`?

Reply:
no, as cleanup jobs are run in parallel after all nodes are rolled out